src/selectors/index.js

```diff
@@ -11,25 +11,15 @@ import * as icons from './icons';
 import * as publish from './publish';
 import * as zippedProfiles from './zipped-profiles';
 
-// Normally we don't need that type because flow can infer types, but on that
-// object, it gives weird flow errors that say missing type annotation for X.
-// Only adding a type like that seems to solve it.
```

Comment:
The true reason is that Flow requires that exported "things" are explicitely typed. I agree that the error message was super weird :)